xbmc/addons/kodi-dev-kit/include/kodi/tools/StringUtils.h

```diff
@@ -0,0 +1,3098 @@
+/*
+ *  Copyright (C) 2005-2020 Team Kodi
+ *  This file is part of Kodi - https://kodi.tv
+ *
+ *  SPDX-License-Identifier: GPL-2.0-or-later
+ *  See LICENSES/README.md for more information.
+ */
+
+#pragma once
+
+#ifdef __cplusplus
+
+#include <algorithm>
+#include <cassert>
+#include <cinttypes>
+#include <cmath>
+#include <cstdarg>
+#include <cstring>
+#include <iomanip>
+#include <string>
+#include <vector>
+
+#include <fstrcmp.h>
```

Comment:
this is libfstrcmp, is this really needed?

Reply:
Have removed it, was gone down that "fstrcmp.h" use a own lib.
Maybe if needed in future it can be added as "C" interface function Kodi (or the addon it himself :smile:)